example/rive_viewer.cpp

```diff
@@ -160,6 +171,44 @@ static void cleanExample()
     delete animationInstance;
 }
 
+static void animPopupItemCb(void *data EINA_UNUSED, Evas_Object *obj, void *event_info)
+{
+   int animationIndex = static_cast<int>(reinterpret_cast<intptr_t>(data));
+   initAnimation(animationIndex);
+   elm_ctxpopup_dismiss(animationPopup);
+}
+
+static Elm_Object_Item* animPopupItemNew(Evas_Object *obj, const char *label, int index)
+{
+   Evas_Object *ic = NULL;
+   Elm_Object_Item *it = NULL;
+
+   if (!obj) return NULL;
+
+   it = elm_ctxpopup_item_append(obj, label, NULL, animPopupItemCb, (void*)index);
+   return it;
+}
+
+static void animPopupDismissCb(void *data EINA_UNUSED, Evas_Object *obj, void *event_info EINA_UNUSED)
+{
+    if (animationPopup) evas_object_del(obj);
```

Comment:
This looks weird condition. 
you compare different instances here. I'd rather suggest you to like this.

evas_object_del(obj);
animationPopup = nullptr;